cluster/gce/gci/configure-helper.sh

```diff
@@ -130,7 +130,7 @@ function main() {
     start-kube-controller-manager
     start-kube-scheduler
     wait-till-apiserver-ready
-    start-workload-controller-manager
+   # start-workload-controller-manager
```

Comment:
can we use this variable to control it as the partitionserver-configure-helper.sh does ?
{ENABLE_WORKLOADCONTROLLER}

Reply:
The issue is that we need additional work to disable rs controller and deployment controller in KCM. This PR is trying to synchronize perf test and master, would rather leave enhancement in separate changes.

Reply:
we do have this parameter, but in this time, it only works for partition servers and it doesn't work for master server. I agree with Ying, for now, we can keep this way.